Watchman.Discord/Areas/Protection/Controllers/MuteUserController.cs

```diff
@@ -1,25 +1,34 @@
 ﻿#nullable enable
+using System.Collections.Generic;
+using System.Linq;
 using Devscord.DiscordFramework.Commons.Exceptions;
 using Devscord.DiscordFramework.Framework.Architecture.Controllers;
 using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
 using Devscord.DiscordFramework.Middlewares.Contexts;
 using Devscord.DiscordFramework.Services;
 using System.Threading.Tasks;
+using Watchman.Discord.Areas.Administration.BotCommands;
+using Watchman.Discord.Areas.Administration.Controllers;
+using Watchman.Discord.Areas.Administration.Models;
 using Watchman.Discord.Areas.Protection.Services;
 
 namespace Watchman.Discord.Areas.Protection.Controllers
 {
     public class MuteUserController : IController
     {
+        private readonly MutingHelper _mutingHelper;
+        private readonly DirectMessagesService _directMessagesService;
         private readonly MutingService _mutingService;
         private readonly UnmutingService _unmutingService;
         private readonly UsersService _usersService;
 
-        public MuteUserController(MutingService mutingService, UnmutingService unmutingService, UsersService usersService)
+        public MuteUserController(MutingService mutingService, UnmutingService unmutingService, UsersService usersService, DirectMessagesService directMessagesService, MutingHelper mutingHelper)
         {
             this._mutingService = mutingService;
             this._unmutingService = unmutingService;
             this._usersService = usersService;
+            this._mutingHelper = mutingHelper;
+            this._directMessagesService = directMessagesService;
```

Comment:
to już takie czepianie sie
ale kolejność powinna być taka jak kolejność parametrów 
więc te 2 odwrotnie